src/plugin/minimap.js

```diff
@@ -124,7 +124,7 @@ export default class MinimapPlugin {
             if (!document.body.contains(this.params.container)) {
                 ws.container.insertBefore(this.params.container, null);
             }
-            // how solve this dependency??
+
             if (this.wavesurfer.regions && this.params.showRegions) {
```

Comment:
Do you remember my question here? If create a custom region plugin and call it in different way (e.g `customRegions` instead of `regions`), this statement will be always false, so the regions in the minimap will not be showed

Reply:
ah yes. something like a new plugin parameter, e.g. `regionsPlugin`, where `this.wavesurfer.regions` is the default, but you should also be able to pass in your custom regions plugin instance (created earlier I guess)..?

Reply:
Yes, maybe something like this:
- Add new `minimap` parameter, e.g `regionsPluginName`
- If is not passed from outside, its default is `'regions'`
- In the statement, check for `regions` plugin in this way:
  ```
  if (this.wavesurfer[this.params.regionsPluginName] && this.params.showRegions) {
       this.regions();
  }
What do you think?

Reply:
Yes, sounds good. But I would add a new public var for `this.wavesurfer[this.params.regionsPluginName]` so it can be easily accessed for other usage.

Reply:
You mean in the `minimap`, correct?

Reply:
This changes have to be included in this pull request?

Reply:
yea in minimap. The ability to use a customized regions plugin in the minimap plugin should/can be done in another pull request.